cowprotocol/cowswap
# Update eslint

# Summary

Update eslint, for some reason now it works. I took the latest version that doesn't change the major.

Created because this PR was nor working locally for me...
https://github.com/cowprotocol/cowswap/pull/499

![image](https://user-images.githubusercontent.com/2352112/166288997-7f94aebb-cfba-49bc-abbf-0875e6154014.png)


I still don't get why GH actions made it work,

# To Test
- download locally and run. 
- See that we don't get the errors any more
